sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/owaspzapwrapper/scan/AbstractScan.java

```diff
@@ -464,9 +466,19 @@ private void renameReportFileIfFileExtensionIsNotJSON() {
         }
     }
 
+    private void visitInclude(String url) {
+        try {
+            String followredirects = "false";
+            clientApi.core.accessUrl(url, followredirects);
+        } catch (ClientApiException e) {
+            LOG.error("Trying to access URL {} produced the error: {}", url, e.getMessage());
```

Comment:
`got…` or `received the error`

```suggestion
            LOG.error("While trying to access URL {} got the error: {}", url, e.getMessage());
```